decidim-results/app/views/decidim/results/results/_share.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="text-center">
```

Comment:
Yes, the first version was dependant on the model but this one is actually shareable.